iib/workers/tasks/iib_static_types.py

```diff
@@ -52,49 +54,90 @@ class BundleImage(TypedDict):
     csvName: Optional[str]
 
 
-class ResolveImagePullspecs(TypedDict):
-    """TypedDict class for typing the DevelopmentConfig class."""
+class BinaryImageConfig(TypedDict):
+    """Type class referencing Binary Image configuration."""
 
-    type: str
+    prod: Dict[str, str]
```

Comment:
I don't think `prod` is the right key name here. It's a string. Let's keep this Dict[str, Dict[str, str]]

Reply:
I removed the class and replaced it with Dict[str, Dict[str, str]]. Pleas check.